Makefile

```diff
@@ -39,7 +39,8 @@ checks:
 run-build: deps checks clean
 	cd Godeps/_workspace/src/github.com/contiv/ && chmod -R 771 contivmodel/ && cd contivmodel/ && ./generate.sh && \
 	cd /opt/gopath/src/github.com/contiv/netplugin && \
-	godep go install -v $(TO_BUILD) 
+	godep go install -v $(TO_BUILD) && \
+	sudo cp contrib/completion/bash/netctl /etc/bash_completion.d/netctl
```

Comment:
+1 on making this part of the release tar; because that would be needed when we make the changes for ansible repo to auto-configure this.